recipes-core/everest/everest-core-rust-crates.inc

```diff
@@ -0,0 +1,239 @@
+           crate://crates.io/tokio-io-timeout/1.2.0 \
+           crate://crates.io/tokio-macros/2.5.0 \
+           crate://crates.io/tokio-stream/0.1.17 \
+           crate://crates.io/tokio-util/0.7.13 \
+           crate://crates.io/tonic/0.8.3 \
+           crate://crates.io/tower/0.4.13 \
+           crate://crates.io/tower-layer/0.3.3 \
+           crate://crates.io/tower-service/0.3.3 \
+           crate://crates.io/tracing/0.1.41 \
+           crate://crates.io/tracing-attributes/0.1.28 \
+           crate://crates.io/tracing-core/0.1.33 \
+           crate://crates.io/tracing-futures/0.2.5 \
+           crate://crates.io/trust-dns-client/0.23.2 \
+           crate://crates.io/trust-dns-proto/0.23.2 \
+           crate://crates.io/try-lock/0.2.5 \
+           crate://crates.io/unicode-bidi/0.3.18 \
+           crate://crates.io/unicode-ident/1.0.16 \
+           crate://crates.io/unicode-normalization/0.1.24 \
+           crate://crates.io/unicode-segmentation/1.12.0 \
+           crate://crates.io/unicode-width/0.1.14 \
+           crate://crates.io/unsafe-libyaml/0.2.11 \
+           crate://crates.io/url/2.5.4 \
+           crate://crates.io/utf16_iter/1.0.5 \
+           crate://crates.io/utf8_iter/1.0.4 \
+           crate://crates.io/utf8parse/0.2.2 \
+           crate://crates.io/want/0.3.1 \
+           crate://crates.io/wasi/0.11.0+wasi-snapshot-preview1 \
+           crate://crates.io/wasm-bindgen/0.2.100 \
+           crate://crates.io/wasm-bindgen-backend/0.2.100 \
+           crate://crates.io/wasm-bindgen-macro/0.2.100 \
+           crate://crates.io/wasm-bindgen-macro-support/0.2.100 \
+           crate://crates.io/wasm-bindgen-shared/0.2.100 \
+           crate://crates.io/winapi-util/0.1.9 \
+           crate://crates.io/windows-core/0.52.0 \
+           crate://crates.io/windows-sys/0.52.0 \
+           crate://crates.io/windows-sys/0.59.0 \
+           crate://crates.io/windows-targets/0.52.6 \
+           crate://crates.io/windows_aarch64_gnullvm/0.52.6 \
+           crate://crates.io/windows_aarch64_msvc/0.52.6 \
+           crate://crates.io/windows_i686_gnu/0.52.6 \
+           crate://crates.io/windows_i686_gnullvm/0.52.6 \
+           crate://crates.io/windows_i686_msvc/0.52.6 \
+           crate://crates.io/windows_x86_64_gnu/0.52.6 \
+           crate://crates.io/windows_x86_64_gnullvm/0.52.6 \
+           crate://crates.io/windows_x86_64_msvc/0.52.6 \
+           crate://crates.io/write16/1.0.0 \
+           crate://crates.io/writeable/0.5.5 \
+           crate://crates.io/yoke/0.7.5 \
+           crate://crates.io/yoke-derive/0.7.5 \
+           crate://crates.io/yore/1.1.0 \
+           crate://crates.io/zerocopy/0.7.35 \
+           crate://crates.io/zerocopy-derive/0.7.35 \
+           crate://crates.io/zerofrom/0.1.5 \
+           crate://crates.io/zerofrom-derive/0.1.5 \
+           crate://crates.io/zerovec/0.10.4 \
+           crate://crates.io/zerovec-derive/0.10.3 \
```

Comment:
I hope this isn't a maintenance nightmare, but I have no experience with Rust recipes. 😅